progress.go

```diff
@@ -92,9 +95,17 @@ func (p *Progress) Report(s Stat) {
 	p.curM.Lock()
 	p.cur.Add(s)
 	cur := p.cur
+	needUpdate := false
+	if time.Since(p.lastUpdate) > minTickerTime {
```

Comment:
That is intentional. `Report()` is called from multiple goroutines all read (and update) `p.lastUpdate`.